lab/src/Microsoft.SyndicationFeed/examples/RssWriteItemWithCustomElementExample.cs

```diff
@@ -41,7 +47,7 @@ class RssWriteItemWithCustomElement
             var content = new SyndicationContent(formatter.CreateContent(item));
 
             // Add custom fields/attributes
-            content.AddField(new SyndicationContent("CustomElement", "My Value"));
+            content.AddField(new SyndicationContent("example:customElement", "http://contoso.com/syndication/feed/examples", "Custom Value"));
```

Comment:
http://contoso.com/syndication/feed/examples should be in a constant

const string EXAMPLE_NS = "http://contoso.com/syndication/feed/examples";